bin/improver-threshold

```diff
@@ -95,6 +97,19 @@ def main():
                         "indicating a narrower fuzzy factor region / sharper "
                         "threshold. NB A fuzzy factor cannot be used with a "
                         "zero threshold or a threshold_config file.")
+    parser.add_argument("--collapse-coord", type=str,
+                        metavar="COLLAPSE-COORD", default="None",
+                        help="An optional ability to set which coordinate "
+                        "we want to collapse over. The default is set "
+                        "to None.")
+    parser.add_argument("--coord_exp_val", metavar="COORD_EXPECTED_VALUES",
```

Comment:
It might be best if `coord_exp_val` and `coordinate_unit` are removed as options, as we're not using these options for collapsing the realization at the moment. The usage of `ChooseDefaultWeightsLinear` can just be changed to not take these arguments, as these are both keyword arguments to the `ChooseDefaultWeightsLinear` process method that don't need to be provided i.e.
```python
ChooseDefaultWeightsLinear(y0val=1.0, slope=0.0).process(result_no_collapse_coord, args.collapse_coord)
```

Reply:
This has been changed now :)